content/docs/start/data-and-model-versioning.md

```diff
@@ -247,6 +247,16 @@ defines data file versions. Git itself provides the version control. DVC in turn
 creates these `.dvc` files, updates them, and synchronizes DVC-tracked data in
 the <abbr>workspace</abbr> efficiently to match them.
 
+## Model versioning
+
+Apart from data files, DVC eases the way you work with models. Models in a
```

Comment:
I don't think "eases" means "makes easy". Let's try to use simple straightforward language like:

```suggestion
DVC helps you handle model files too. Models in a
```

Reply:
Although maybe we don't need this sentence? (See https://github.com/iterative/dvc.org/pull/2214#pullrequestreview-593674216). Thanks